src/client/pages/Offer/Introduction/Sidebar/index.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useState, useCallback } from 'react'
```

Comment:
This is a copy of `OfferNew/Introduction/Sidebar/index.tsx` with some changes:
- updated component API
    _Old_:
    ```typescript
     type Props = {
         offerData: OfferData
          refetchOfferData: () => Promise<void>
          onCheckoutOpen: () => void
     }
    ```
   _New_:
   ```typescript
    export type SidebarProps = {
        quoteCartId: string
        offerData: OfferData
        refetchOfferData: () => Promise<void>
        onCheckoutOpen: () => void
    }
   ```
- make used of _QuoteCart API_ to add/remove _Campaign Codes_